src/components/developers/developers.styles.js

```diff
@@ -53,6 +53,18 @@ const useDevelopersStyles = createUseStyles(theme => ({
       textAlign: 'left'
     }
   },
+  section1img: {
+    backgroundImage: 'url(./technology-section1.png)',
+    backgroundRepeat: 'no-repeat',
+    backgroundPosition: 'center',
+    backgroundSize: 'contain',
+    margin: 'auto',
+    width: '100%',
+    height: '237px',
```

Comment:
theme.spacing? here and elsewhere

Reply:
I don't thing so, since these are explicit sizes of the graphics. Chami insisted that these are exact size as in Zeplin, so I think we should not tie them to spacers.

Reply:
I would use `maxWidth` instead, because if the screen is wider than `px`, we will have a problem hehe But we can improve this later on

Reply:
Width is %, and I was thinking the same for `height` vs. `maxHeight` but at the end I found no difference. But yes we can take a look at it later on.